packages/backend/src/api-impl.ts

```diff
@@ -54,7 +54,11 @@ export class BootcApiImpl implements BootcApi {
     return checkPrereqs(await getContainerEngine());
   }
 
-  async checkVMLaunchPrereqs(build: BootcBuildInfo): Promise<string | undefined> {
+  async checkVMLaunchPrereqs(buildId: string): Promise<string | undefined> {
+    const build = this.history.getHistory().find(build => build.id === buildId);
+    if (!build) {
+      return Promise.reject(new Error(`Could not find build: ${buildId}`));
```

Comment:
question : why promise.rejectand not throw new Error ?

Reply:
Should be throw error, fixed in new commit.